packages/boxel/tests/dummy/config/deprecation-workflow.js

```diff
@@ -8,7 +8,7 @@ self.deprecationWorkflow.config = {
     { handler: 'silence', matchId: 'ember-modifier.function-based-options' },
     { handler: 'silence', matchId: 'ember-modifier.no-args-property' },
     { handler: 'silence', matchId: 'ember-modifier.no-element-property' },
-    { handler: 'silence', matchId: 'ensure-safe-component.string' },
```

Comment:
I was removing the lines in my PRs, should I be using `throw` instead? 😯

Reply:
I think so. It will prevent us from backsliding.